### ISSUE TITLE
NVDA can not read text in dropdown menu when the mouse hovers it, but it works when keyboard focuses on it

### ISSUE THREAD

**Issue by eileencai** [nvaccess/nvda#16117]
### Steps to reproduce:
1. run FortiClient 7.2.3.GA 
2. use the mouse to expand the dropdown menu, hovers any option of the menu
3. NVDA reads "vpn name list 'ipsec-ems' expanded'" then keeps silent, when you hovers any option, it does not read the option any more

### Actual behavior:
NVDA reads "vpn name list 'ipsec-ems' expanded'" then keeps silent, when you hovers any option, it does not read the option any more

### Expected behavior:
NVDA is expected to read the option value when the mouse hovers it

### NVDA logs, crash dumps and other attachments:
The HTML is as below:
      <label for="vpn-connection" id="vpn-connection-label" class="vpn-label margin-right font-heavy off">{{vpnName}}</label>
      <select id="vpn-connection" role="listbox" class="select lg vpn-select" aria-labelledby="vpn-connection-label">
        <optgroup label="Corporate VPNs">
          <option value="ipsec-ems" aria-label="ipsec-ems" data-type="ipsec" data-corporate="true">ipsec-ems
          </option>
          <option value="sslvpn_redundant_ems" aria-label="sslvpn_redundant_ems" data-type="ssl" data-corporate="true" selected="selected">vpn_ems
          </option>
        </optgroup>
      </select>

#### NVDA installed/portable/running from source:
official website (https://www.nvaccess.org/download/)

#### NVDA version:
2023.3.0.29780

#### Windows version:
Windows 10 Pro

#### Name and version of other software in use when reproducing the issue:
FortiClient 7.2.3.GA

#### Does the issue still occur after restarting your computer?
yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no

#### If NVDA add-ons are disabled, is your problem still occurring?
no add-ons

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes

**Comment by Adriani90** [comment]
@eileencai how is the css of those objects?
You posted only a small snippet here, please provide a minimal test case, maybe a small landing page designed with Forticlient where we can test this. Otherwise it is really hard to investigate further.

**Comment by Adriani90** [comment]
No updates upon request since more than one year, minimal test case is missing. Closing as abandoned. If you are still having this issue, please comment with updated steps to reproduce and minimal test canse, and then we can reopen.
